vrecord

```diff
@@ -1570,7 +1662,7 @@ if [[ "${RUNTYPE}" = "record" ]] ; then
     [f]astats=metadata=1:reset=1,adrawgraph=m1=lavfi.astats.2.Min_level:m2=lavfi.astats.2.Max_level:size=700x160:bg=Black:fg1=0xFF00FF00:fg2=0xFF00FF00:slide=scroll:min=-32767:max=32767[wav2]"
     AP_MAP='[c1][wav1][wav2]xstack=inputs=3:layout=0_0|0_h0|0_h0+h1[out0]'
 else
-    PRINT_PHASE=",drawtext=fontfile=${DEFAULTFONT}:box=1: text=Phase\\\: %{metadata\\\:lavfi.aphasemeter.phase}:x=135:y=380:fontcolor=black"
+    PRINT_PHASE=",drawtext=fontfile=${DEFAULTFONT}:box=1: text='Phase\: %{metadata\\\:lavfi.aphasemeter.phase}':x=135:y=380:fontcolor=black"
```

Comment:
Changes to single quotes seem to have broken text substitution on my end (Ubuntu) 
![image](https://user-images.githubusercontent.com/12941699/95108089-a4fe1f00-06ef-11eb-8220-7310f47344eb.png)